app/assets/stylesheets/components/_navbar.scss

```diff
@@ -10,3 +10,35 @@
 .navbar-lewagon .navbar-brand img {
   width: 40px;
 }
+
+.nav-item {
+  margin-right: 40px;
+}
+
+#logo-text {
+  position: absolute;
+  left: 80px;
+  text-shadow:  0.5px 0.5px #e49999;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.nav-item-dropdown {
+ border-radius: 50%;
+}
+
+// IN CASE THIBAULT DOES NOT LIKE BOOTSTRAP
+// .nav-link {
+//   background-color: #008CBA;
+//   border: none;
+//   border-radius: 5%;
+//   color: white;
+//   padding: 15px 32px;
+//   text-align: center;
+//   text-decoration: none;
+//   display: inline-block;
+//   font-size: 16px;
+// }
```

Comment:
comments can be removed, using bootstrap is good for responsive design

Reply:
done